Epsilon/Components/LearningDomainComponentManager.cs

```diff
@@ -0,0 +1,24 @@
+using Epsilon.Abstractions.Components;
+using Epsilon.Abstractions.Services;
+
+namespace Epsilon.Components;
+
+public record LearningDomainComponentManager : IComponentManager
```

Comment:
This isn't a record, but a class.

And what is a learning domain component? It's abstract, so you cannot construct it, aka you don't know what this component is yet. I'm not sure if this is part of the draft or if this is actually something you wanted to include.

Reply:
@Typiqally LearningDomainComponent should be the word (/other formats) component that shows data the same way the performance dashboard does so. The component itself is not realised yet because this will be a task for either Saamie or Bas. Is that what you meant? Name is a work in progress btw

Reply:
So it is the CompetenceProfile component is what you're saying? Basically the "matrix" we have as the perf. dashboard.

Reply:
Yeah, the name is trash, ill change that as well. Implementing that component, and the kpimatrixoutcomes component file structure rn. althought the second name also needs to be changed

Reply:
This was the old wiki regarding component names, we agreed on them then so just use these please: https://github.com/Typiqally/epsilon/wiki/Components

Reply:
Is KPI still ok to use? isn't that a bit domain specific?

Reply:
I think that's a fair point. In any case, KPI's are just outcomes, which will be the case in any domain context. So maybe just call it the OutcomeMatrix or smth along those lines.